recipe/meta.yaml

```
@@ -29,10 +29,15 @@ outputs:
         - git
         - cmake
         - make  # [not win]
+        - htslib >=1.15  # [not win]
```

Shouldn't this only be required when cross-compiling, ie the osx-arm64 build?

```suggestion
        - htslib >=1.15  # [build_platform != target_platform]
```

---

I think yes, and also it only needs to be in host requirements.